### PR TITLE
Fixed versions of dependencies

### PR DESCRIPTION
**Pull request by emreathelix**
Setting the dependency versions to make the documentation of 3rd party libraries easier for the FDA project.

### CODE REVIEW

File: providers/reported_at/provider_test.go [hunk 1]
```diff
@@ -44,5 +44,5 @@ func TestReportedAtFiltering(t *testing.T) {
 	setup(t, RecommendedConfig)
 
 	testProvider.Info(context.Background(), false, "foo")
-	Expect(output.String()).To(MatchRegexp(`time=sometime level=info msg=foo reportedAt=".*go/src/testing/testing.go:\d+`))
+	Expect(output.String()).To(MatchRegexp(`time=sometime level=info msg=foo reportedAt=".*go/.*src/testing/testing.go:\d+`))
```
Comment (emreathelix) [inline providers/reported_at/provider_test.go]: Had to make this change because the tests were failing with the message:

```
Expected
            <string>: time=sometime level=info msg=foo reportedAt="/usr/local/Cellar/go/1.11/libexec/src/testing/testing.go:830"

        to match regular expression
            <string>: time=sometime level=info msg=foo reportedAt=".*go/src/testing/testing.go:\d+
```

Comment (gir) [reply providers/reported_at/provider_test.go]: Nit: Just get rid of the extra `.*go/` part of the regex, it seems pretty useless.
```suggestion
	Expect(output.String()).To(MatchRegexp(`time=sometime level=info msg=foo reportedAt=".*src/testing/testing.go:\d+`))
```

File: providers/reported_at/provider_test.go [hunk 2]
```diff
@@ -44,5 +44,5 @@ func TestReportedAtFiltering(t *testing.T) {
 	setup(t, RecommendedConfig)
 
 	testProvider.Info(context.Background(), false, "foo")
-	Expect(output.String()).To(MatchRegexp(`time=sometime level=info msg=foo reportedAt=".*go/src/testing/testing.go:\d+`))
+	Expect(output.String()).To(MatchRegexp(`time=sometime level=info msg=foo reportedAt=".*src/testing/testing.go:\d+`))
```
Comment (the-gigi) [inline providers/reported_at/provider_test.go]: is that related?

Comment (emreathelix) [reply providers/reported_at/provider_test.go]: I couldn't get the tests to pass without this

Comment (emreathelix) [reply providers/reported_at/provider_test.go]: This is what I was getting without this change

```
Expected
      <string>: time=sometime level=info msg=foo reportedAt="/usr/local/Cellar/go/1.11/libexec/src/testing/testing.go:830"

to match regular expression
      <string>: time=sometime level=info msg=foo reportedAt=".*go/src/testing/testing.go:\d+
```

Comment (the-gigi) [reply providers/reported_at/provider_test.go]: 👍
